src/stream/Encryption.ts

```diff
@@ -28,36 +33,75 @@ class InvalidGroupKeyError extends Error {
     }
 }
 
-export class GroupKey {
+type GroupKeyObject = {
+    id: string,
+    hex: string,
+    data: Uint8Array,
+}
+
+type GroupKeyProps = {
+    groupKeyId: string,
+    groupKeyHex: string,
+    groupKeyData: Uint8Array,
+}
+
+function GroupKeyObjectFromProps(data: GroupKeyProps | GroupKeyObject) {
```

Comment:
Is either `GroupKeyObject` or `GroupKeyProps` a deprecated form or group key definition object? Or do we need both for other reasons?

Reply:
I don't recall the details but the object is in different shapes depending on the source.

Reply:
If the are equal variants I could even name those types as `GroupKeyDefinition1` and `GroupKeyDefinition2`. But doesn't make big difference, any naming is fine.